qiskit_quantuminspire/api/client.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Optional
+
+import compute_api_client
+
+from qiskit_quantuminspire.api.authentication import IdentityProvider, OauthDeviceSession
+from qiskit_quantuminspire.api.settings import ApiSettings
+
+
+class Configuration(compute_api_client.Configuration):  # type: ignore[misc]
```

Comment:
Please add doctrings here and at other places

Reply:
Done